tools/cabana/cabana.cc

```diff
@@ -22,12 +24,22 @@ int main(int argc, char *argv[]) {
     cmd_parser.showHelp();
   }
 
+  QString uuid =  QUuid::createUuid().toString(QUuid::WithoutBraces);
```

Comment:
let's put this helper in `common/`

Reply:
do you mean cabana/common or somewhere else such as /ui/qt/util.cc?

Reply:
the root common/

Reply:
I have reverted https://github.com/commaai/openpilot/pull/26286/commits/19ef28945e8647724bbfc6c8469a664a58c09f16 , how about keep this pr as it is ,and move to common next time?

Reply:
sure, but I meant make a common function to setup a prefix environment